### PR TITLE
Release 17.0

### PR DESCRIPTION
**Pull request by scottilee**
#### What this PR does / why we need it:
GA release 17.0

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# v17.17.0
```
Comment (roycaihw) [inline CHANGELOG.md:1]: Did you pull the master branch? I thought https://github.com/kubernetes-client/python/pull/1455 would be included

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,12 +29,6 @@ Changelog since v17.14.0a1:
 
 Kubernetes API Version: 1.17.14
 
-**Important Information:**
-
-- The Kubernetes Python client versioning scheme has changed. The version numbers used till Kubernetes Python Client v12.y.z lagged behind the actual Kubernetes minor version numbers. From this release, the client is moving a version format `vY.Z.P` where `Y` and `Z` are respectively from the Kubernetes version `v1.Y.Z` and `P` would incremented due to changes on the Python client side itself. Ref: https://github.com/kubernetes-client/python/issues/1244
-- Python 2 had reached [End of Life](https://www.python.org/doc/sunset-python-2/) on January 1, 2020. The Kubernetes Python Client will drop support for Python 2 from the next release (v18.0.0) and will no longer provide support to older clients as per the [Kubernetes support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions).
```
Comment (roycaihw) [inline CHANGELOG.md:26]: I think we don't need to remove the old notification. It's fine to re-emphasize it

Comment (scottilee) [reply CHANGELOG.md:26]: Will add it back.

File: CHANGELOG.md (hunk 1 above)
Comment (scottilee) [reply CHANGELOG.md:1]: No will pull it with `git pull -X theirs upstream master`.

**Pull request closed without merging**